i3pystatus/openvpn.py

```diff
@@ -54,7 +54,13 @@ class OpenVPN(IntervalModule):
     def init(self):
         if not self.vpn_name:
             raise Exception("vpn_name is required")
-
+        
+        # use new service names if openvpn version is 2.4 (2.5 is not out yet...)
+        if run_through_shell('openvpn --version | grep -i "openvpn 2.4"', True).out.strip():
```

Comment:
This is a good idea, but I think it should be an option. People might set `vpn_up_command` and `vpn_down_command` to execute scripts instead of openvpn directly and this would clobber their configuration.